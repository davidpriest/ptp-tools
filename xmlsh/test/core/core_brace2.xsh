# core_brace2.xsh
# Testing more complicated brace syntax

# Test case - was failing on 9/17/2007]
# <version build="20090917 1037" relsase="b1_20090917"/>
for a in a b c ; do
echo In for 
{
 	echo Value is $a
}
done