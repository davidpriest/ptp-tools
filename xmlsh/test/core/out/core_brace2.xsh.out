In for
Value is a
In for
Value is b
In for
Value is c
